spec/views/hq/orphans/show_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe "hq/orphans/show.html.haml", type: :view do
+  let(:orphans) { FactoryGirl.build_stubbed(:orphan_list) }
```

Comment:
The orphan factory builds the `orphan_list` association. Do you need to create it here explicitly?